tnz/zti.py

```diff
@@ -115,6 +116,11 @@ def __init__(self, stdin=None, stdout=None):
         if os.getenv("ZTI_AUTOSIZE"):
             self.autosize = True
 
+        self.macros_dir = os.getenv("ZTI_MACROS_DIR")
```

Comment:
There seems to be no reason to make `macros_dir` a property of Zti. I suggest just doing the environment variable inspection in `__register_macros()`. If you would rather do the environment variable inspection here... then pass `macros_dir` into `__register_macros()`.